syntax/std_seq_bytes_helper.go

```diff
@@ -28,6 +28,9 @@ func bytesSplit(delimiter rel.Value, subject rel.Bytes) rel.Value {
 		splitted = strings.Split(subject.String(), delimiter.String())
 	case rel.GenericSet:
 		splitted = strings.Split(subject.String(), mustAsString(delimiter))
+	default:
+		panic(`the delimiter of api //seq.split(delimiter, subject) is not an array of byte, ` +
+			`it can try <<delimiter>> to generate byte array`)
```

Comment:
It needs to mention the type of the delimiter
```suggestion
		panic(fmt.Sprintf("delimiter and subject have to be of the same type, currently: delimiter: %T, subject: %T", delimiter, subject))
```

needs to import package `fmt` before merging that